alarm_control_panel-card/alarm_control_panel-card.js

```diff
@@ -136,21 +137,21 @@ class AlarmControlPanelCard extends HTMLElement {
       ${(entity.attributes.code_format == 'Number' || config.force_keypad)? `
           <div class="pad">
             <div>
-              <paper-button data-digit="1" raised>1</paper-button>
-              <paper-button data-digit="4" raised>4</paper-button>
-              <paper-button data-digit="7" raised>7</paper-button>
+              ${this._generateDigitButton(1, "<br/>", config.display_letters)}
```

Comment:
config.display_letters should be available with this._config in any method. No need to send it as parameter

Reply:
Sorry, I'm a clumsy C++ dev just poking at this stuff ;)

Reply:
No worries. I appreciate the help a lot!